churn_prediction/data/interim/interm_data_process.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+
+class DataPreprocessing:
+    
+    def __init__(self, train_data, transaction_data, user_logs_data, members_data) -> None:
+        self.train_data = train_data
+        self.transaction_data = transaction_data
+        self.user_logs_data = user_logs_data
+        self.members_data = members_data
+
+    def drop_duplicates(self):
```

Comment:
Add the same param information and doc string to every method and add error handling blocks as well and use rich print for that. https://rich.readthedocs.io/en/stable/introduction.html

Reply:
(rich print)so basically you want to print params of every class or method in CLI

Reply:
Yes.